extensions/impl/kafka/sink.go

```diff
@@ -161,6 +161,9 @@ func (k *KafkaSink) Provision(ctx api.StreamContext, configs map[string]any) err
 	if err != nil {
 		return err
 	}
+	if k.kc.BatchSize == 0 && k.kc.LingerInterval == 0 {
```

Comment:
How about just report error?

Reply:
`batchSize:0 ` and `lingerInterval:0` will make batch op can't send the msg, and the `batchSize:0` will be sent in the props by neuronex by default if it is not set .